packages/use-wallet/src/wallets/liquid.ts

```diff
@@ -139,6 +139,7 @@ export class LiquidWallet extends BaseWallet {
       this.logger.debug('Signing transactions...', { txnGroup, indexesToSign })
 
       const authClient = this.authClient || (await this.initializeClient())
+      // @ts-expect-error - TODO: update liquid-auth-use-wallet-client to use algosdk v3
```

Comment:
@HashMapsData2Value The `liquid-auth-use-wallet-client` package will need to support `algosdk` v3 in the next major release. The `Transaction` interfaces are incompatible between v2 + v3.